float64 timeErrorToFirstPose
float64 timeErrorToClosestPose
float64 poseError
int32 closestWP